module-2/worksheet/2-your-node-in-a-mesh-network.md

```diff
@@ -0,0 +1,91 @@
+Your node in a mesh network
+===========================
+
+## Connecting to your Raspberry Pi
+
+1. Verify that the Raspberry Pi has the SD card in place. Power on the device and wait for a solid green light with a flashing red light.
+
+1. Connect your computer's WiFi Client to the Raspberry Pi's WiFi Access Point:
+
+    - SSID: `<hostname>`
+    - Password: `password`
+
+1. Once your computer is connected to the WiFi Access Point, you can access the Raspberry Pi via a Secure Shell (SSH):
+
+    - Host: `<hostname>.local` (or `10.0.0.1`)
+    - Username: `root`
+    - Password: `root`
+
+    On **macOS or Linux** using **Terminal**, enter `ssh root@example.local` followed by the password `root`.
+
+    On **Windows** using the **PuTTY** graphical interface, enter the `Host` and select `SSH` then click `Open` to initiate an SSH session. Enter the `Username` and `Password` when prompted.
+
+You are now connected to and able to run commands from your Raspberry Pi, but unlike last week, your Raspberry Pi is currently not connected to all the other ones in the room. Find a partner and we will manually get the two Raspberry Pis to talk over an ethernet cable. For the rest of this activity, we will use `example` to represent one node, and `example2` to represent the other.
+
+## Making a wired ethernet link
+
+1. The Raspberry Pi has one single ethernet port for wired networking. In the operating system, it is represented by `eth0`. Look at the `eth0` ethernet interface using:
+
+    **root@example:~#** `networkctl status eth0`
+
+    You should see `State: no-carrier (configuring)`. This is because the ethernet port is not connected to anything. We need to build a road between the two Raspberry Pis.
+
+1. Connect the two nodes with an ethernet cable, then run the same command again. You will find that the state has changed to `State: degraded (configuring)`. Even though we have a road (i.e. ethernet cable), there is no street addresses (i.e. IP addresses) or road signs for directions (i.e. routes).
+
+1. Assign an IP address to `eth0` and add a route on `example`:
+
+    **root@example:~#** `ip addr add 192.168.0.1 dev eth0`
+    **root@example:~#** `ip route add 192.168.0.0/24 via 192.168.0.1`
```

Comment:
Specify subnet in `ip addr add` as @darkdrgn2k [suggested](https://github.com/benhylau/mesh-workshop/issues/5):

```
root@bathurst:~# ip addr add 192.168.0.1/24 dev eth0
root@bathurst:~# ip route
10.0.0.0/24 dev wlan0 proto kernel scope link src 10.0.0.1
172.17.0.0/16 dev docker0 proto kernel scope link src 172.17.0.1 linkdown
192.168.0.0/24 dev eth0 proto kernel scope link src 192.168.0.1
```